source/connect/tls.txt

```diff
@@ -0,0 +1,271 @@
+.. c-tls:
+
+========================================
+Configure Transport Layer Security (TLS) 
```

Comment:
Confirming: is https://www.mongodb.com/docs/languages/c/c-driver/current/libmongoc/guides/configuring_tls/ planned to be removed?

Reply:
yup – the docs are being standardized to the layout that other drivers docs follow, like the [pymongo docs](https://www.mongodb.com/docs/languages/kotlin/kotlin-sync-driver/current/indexes/), so this page will replace the previous 'configuring TLS' page